iothub/service/src/ServiceClient.cs

```diff
@@ -99,111 +100,75 @@ public static ServiceClient CreateFromConnectionString(string connectionString,
         /// <summary>
-        /// Removes all messages from a device's queue.
-        /// </summary>
-        /// <param name="deviceId"></param>
-        /// <param name="cancellationToken"></param>
-        /// <returns></returns>
-        public abstract Task<PurgeMessageQueueResult> PurgeMessageQueueAsync(string deviceId, CancellationToken cancellationToken);
-
-        /// <summary>
-        /// Get the FeedbackReceiver
+        /// Get the FeedbackReceiver.
         /// </summary>
         /// <returns>An instance of the FeedbackReceiver</returns>
         public abstract FeedbackReceiver<FeedbackBatch> GetFeedbackReceiver();
 
         /// <summary>
-        /// Get the FeedbackReceiver
+        /// Get the FileNotificationReceiver
         /// </summary>
-        /// <returns>An instance of the FeedbackReceiver</returns>
+        /// <returns>An instance of the FileNotificationReceiver</returns>
         public abstract FileNotificationReceiver<FileNotification> GetFileNotificationReceiver();
 
         /// <summary>
         /// Gets service statistics for the Iot Hub.
         /// </summary>
-        /// <returns>returns ServiceStatistics object containing current service statistics</returns>
-        public abstract Task<ServiceStatistics> GetServiceStatisticsAsync();
-
-        /// <summary>
-        /// Gets service statistics for the Iot Hub.
-        /// </summary>
-        /// <param name="cancellationToken">
-        /// The token which allows the the operation to be cancelled.
-        /// </param>
-        /// <returns>returns ServiceStatistics object containing current service statistics</returns>
-        public abstract Task<ServiceStatistics> GetServiceStatisticsAsync(CancellationToken cancellationToken);
-
-        /// <summary>
-        /// Interactively invokes a method on device
-        /// </summary>
-        /// <param name="deviceId">Device Id</param>
-        /// <param name="cloudToDeviceMethod">Device method parameters (passthrough to device)</param>
-        /// <returns>Method result</returns>
-        public abstract Task<CloudToDeviceMethodResult> InvokeDeviceMethodAsync(string deviceId, CloudToDeviceMethod cloudToDeviceMethod);
-
-        /// <summary>
-        /// Interactively invokes a method on device
-        /// </summary>
-        /// <param name="deviceId">Device Id</param>
-        /// <param name="cloudToDeviceMethod">Device method parameters (passthrough to device)</param>
-        /// <param name="cancellationToken">Cancellation Token</param>
-        /// <returns>Method result</returns>
-        public abstract Task<CloudToDeviceMethodResult> InvokeDeviceMethodAsync(string deviceId, CloudToDeviceMethod cloudToDeviceMethod, CancellationToken cancellationToken);
+        /// <param name="cancellationToken">A cancellation token to cancel the operation.</param>
+        /// <returns>The service statistics that can be retrieved from IotHub, eg. the number of device connected to the hub.</returns>
```

Comment:
IoT hub

Reply:
the number of device(s) - plural

Reply:
connection to the IoT hub

Reply:
or IoT Hub?

Reply:
From what I've heard in the past, we use IoT hub.

Reply:
Interesting, would love to know the reasoning. 
I see IoT **H**ub in many other places in comments in this PR, would like to see it consistent whatever we decide.

Reply:
I know it is inconsistent but from previous PRs feedbacks, I understand that IoT hub is what makes sense. I guess we should decide as a team which to pick. I'm not an expert in this stuff. So would not mind some other opinions.